src/qt/forms/researcherwizardemailpage.ui

```diff
@@ -0,0 +1,90 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ui version="4.0">
+ <class>ResearcherWizardEmailPage</class>
+ <widget class="QWizardPage" name="ResearcherWizardEmailPage">
+  <property name="geometry">
+   <rect>
+    <x>0</x>
+    <y>0</y>
+    <width>630</width>
+    <height>480</height>
+   </rect>
+  </property>
+  <property name="sizePolicy">
+   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
+    <horstretch>0</horstretch>
+    <verstretch>0</verstretch>
+   </sizepolicy>
+  </property>
+  <property name="windowTitle">
+   <string>BOINC Email Address</string>
+  </property>
+  <property name="title">
+   <string>BOINC Email Address</string>
+  </property>
+  <property name="subTitle">
+   <string>Confirm the email address that you provided to register the accounts for BOINC projects that you participate in. Gridcoin uses the email address to detect BOINC projects on your computer.</string>
```

Comment:
This also can be simplified
```suggestion
   <string>Enter the email address you use for your account on BOINC projects. Gridcoin uses it to find BOINC projects on your computer.</string>
```

Reply:
Same thing here. I am with @cyrossignol on this. It is clearer to repeat the subject.

Reply:
I guess that could still work with the other simplifications
```suggestion
   <string>Enter the email address you use for your account on BOINC projects. Gridcoin uses this email to find BOINC projects on your computer.</string>
```

Reply:
I am ok with that I think.

Reply:
I'm going with this:

```suggestion
   <string>Enter the email address that you use for your BOINC project accounts. Gridcoin uses this email address to find BOINC projects on your computer.</string>
```